pldoc/static/js/colors.json

```diff
@@ -0,0 +1,174 @@
+[{
```

Comment:
If using requireJS to import this json file doesn't work, it should also work to convert this into a `*.js` file that just does:

```javascript
module.exports = {
  your: 'json',
  goes: 'here'
};
```

and I can't see any reason requirejs wouldn't handle that fine. As a bonus, then it would get bundled into all the other Javascript, and you wouldn't need to add the Gulp tasks for JSON files you added above.